oraclizeAPI_0.4.25.sol

```diff
@@ -362,157 +399,1290 @@ contract usingOraclize {
         return false;
     }
 
-    function __callback(bytes32 myid, string result) public {
+    function __callback(
+        bytes32 myid,
+        string result
+    )
+        public
+    {
         __callback(myid, result, new bytes(0));
     }
-    function __callback(bytes32 myid, string result, bytes proof) public {
-      return;
-      // Following should never be reached with a preceding return, however
-      // this is just a placeholder function, ideally meant to be defined in
-      // child contract when proofs are used
-      myid; result; proof; // Silence compiler warnings
-      oraclize = OraclizeI(0); // Additional compiler silence about making function pure/view. 
-    }
 
-    function oraclize_getPrice(string datasource) oraclizeAPI internal returns (uint){
-        return oraclize.getPrice(datasource);
+    function __callback(
```

Comment:
i believe this placeholder doesn't work in 0.5 as we want anymore, remember user reporting warning (I know this is 0.4.25 but mentioining here still).

Reply:
So do both of the `__callback` functions need to disappear form the 0.5 version of the API?

Reply:
No we've decided to keep them with whatever was needed to silence them